base/io.jl

```diff
@@ -511,10 +506,7 @@ line.
 
 # Examples
 ```jldoctest
-julia> open("my_file.txt", "w") do io
-           write(io, "JuliaLang is a GitHub organization.\\nIt has many members.\\n");
-       end
-57
+julia> write(io, "JuliaLang is a GitHub organization.\\nIt has many members.\\n");
```

Comment:
I agree very much with your PR, but the append option seems not to be available for the suggested example. `write(filename::AbstractString, ...)` defined in `base/io.jl` can only do `write` and not append ("a").

To solve this, maybe keep the old one and also mention the one-liner?

Reply:
I am not sure what you mean (these examples don't append, right?), but I made two errors that I fixed in my commit just now, maybe that caused some confusion?

Reply:
I meant how do users know how to append to a file

Reply:
Sorry. My bad. This PR only replaces the initial write for some examples and it doesn't change the docstring for `write` itself